app/controllers/challenges_controller.rb

```diff
@@ -14,4 +14,8 @@ def show
     end
   end
 
+  def drawing
+    @challenge = Challenge.find(params[:id])
```

Comment:
Can we add a `challenges_controller_spec.rb` file for this? It shouldn't be anything too crazy - mainly to test that the route / etc works – something as simple as `users_controller_spec` for the show action would be sufficient. I think this also may need to be `Challenge.find(params[:challenge_id])` per your commit message?